pynamodb/tests/test_model.py

```diff
@@ -1477,6 +1477,40 @@ def test_index_count(self):
             }
             deep_eq(args, params, _assert=True)
 
+    def test_index_multipage_count(self):
```

Comment:
Do you mean when `LastEvaluatedKey` is None on the first page? If so, this is covered by the previous test